bindings/nodejs/examples/how_tos/accounts_and_addresses/consolidate-outputs.ts

```diff
@@ -29,23 +29,21 @@ async function run() {
             storagePath: process.env.WALLET_DB_PATH,
         });
 
-        const account = await wallet.getAccount('Alice');
-
         // To create an address we need to unlock stronghold.
         await wallet.setStrongholdPassword(
             process.env.STRONGHOLD_PASSWORD as string,
         );
 
-        // Sync account to make sure account is updated with outputs from previous examples
-        account.sync();
-        console.log('Account synced');
+        // Sync wallet to make sure wallet is updated with outputs from previous examples
+        wallet.sync();
```

Comment:
i know it was like this, but missing a sync 2x

Reply:
Added